src/main/java/org/generationcp/middleware/dao/GermplasmSearchDAO.java

```diff
@@ -193,17 +173,22 @@
             query.addScalar(GermplasmSearchDAO.METHOD_NAME);
             query.addScalar(GermplasmSearchDAO.LOCATION_NAME);
 
-            for (final String propertyId : germplasmSearchParameter.getAddedColumnsPropertyIds()) {
-                //Append _COL if MALE_PARENT_ID, to avoid duplicate column name on select and fetch the correct value for MGID
-                String scalar = propertyId.equals(GermplasmSearchDAO.MALE_PARENT_ID) ? propertyId + "_COL" : propertyId;
-                query.addScalar(scalar);
+            //Excluding Parents Property from SQL
+            final List<String> germplasmTreeNode = Arrays.asList(GermplasmSearchDAO.MALE_PARENT_ID,
```

Comment:
You can make this list a static class variable e.g.
`	private static final List<String> GERMPLASM_TREE_NODE_PROPERTY_IDS = Collections.unmodifiableList(Arrays.asList(GermplasmSearchDAO.MALE_PARENT_ID, ...));`